platforms/tests.py

```diff
@@ -0,0 +1,3 @@
+from django.test import TestCase
+
+# Create your tests here.
```

Comment:
jak będziesz rebasował na developa to może jakiś 1/2 testy np. do tego o czym wspomniał @Tumilok że mu błąd wyrzuca